_data/ArpanSriv/ArpanSriv.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Arpan Srivastava 
+institution: Symbiosis Institute of Technology, Pune 🚩 
```

Comment:
Remove the 🚩(flag) as it is a non-ascii character.

Reply:
Oh! I thought it was a part of the template. Poor me.